css/index.css

```diff
@@ -1,3 +1,172 @@
-/* Use your own code or past solution for Great Idea Web Page CSS here! */
+/* Desktop CSS here! */
+* {
+    box-sizing: border-box;
+}
 
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
 
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+
+body .container {
+	margin: 0 auto;
+	max-width: 880px;
+}
+
+.navigation {
+	  display: flex;
+		flex-direction: row;
+		padding-top: 15px;
+		padding-bottom: 15px;
+		justify-content: space-between;
+		align-items: center;
+}
+
+nav a {
+	color: grey;
```

Comment:
I would suggest making the nav element a flexbox for justifying the `<a>` contents instead and giving it a certain width (and removing the `margin-right` setting for the `<a>` elements inside it) - it will prevent the links from wrapping when the browser width gets smaller.

When you do your media query for mobile view, you can then change the `flex-direction` to column and add some border properties.